frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/utils/utils.test.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  getTextResourcesForLanguage,
+  createTextResourceWithLanguage,
+  convertTextResourceToMutationArgs,
+} from './utils';
+import type { TextResources, TextResourceWithLanguage } from '@studio/content-library';
+import type { TextResource } from '@studio/components';
+import type { UpsertTextResourceMutation } from 'app-shared/hooks/mutations/useUpsertTextResourceMutation';
+
+describe('utils functions', () => {
+  describe('getTextResourcesForLanguage', () => {
+    it('should return undefined if text resources input variable is undefined', () => {
+      const language = '';
+      const textResources = undefined;
+      expect(getTextResourcesForLanguage(language, textResources)).toBeUndefined();
+    });
+
+    it('should return array of text resources with correct input', () => {
+      const language = 'nb';
+      const textResource: TextResource = {
+        id: 'some-id',
+        value: 'some-value',
+      };
```

Comment:
I suggest moving these values to the top of the file. They are used several places in the file, and the same mock data can be used several times 😄 What do you think?